lib/zenaton/services/serializer.rb

```diff
@@ -57,12 +68,7 @@ def decode(json_string)
       private
 
       def basic_type?(data)
-        data.is_a?(String) \
-          || data.is_a?(Integer) \
-          || data.is_a?(Float) \
-          || data.is_a?(TrueClass) \
-          || data.is_a?(FalseClass) \
-          || data.nil?
+        data.class.ancestors.any? { |klass| BASIC_TYPES.include?(klass) }
```

Comment:
When adding `Symbol` to the list, Rubocop was not happy. This slight refactor should please our community linter.